openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -495,7 +495,7 @@ def test_login_user_info_cookie(self):
 
         # Check that the URLs are absolute
         for url in user_info["header_urls"].values():
-            assert 'http://testserver/' in url
+            assert 'http://' in url
```

Comment:
why don't we use the domain anymore?

Reply:
we don't use the domain because some urls in the header have been changed. (account and profile urls)